sovryn-token-bridge/bridge/contracts/Bridge.sol

```diff
@@ -500,6 +509,15 @@ contract Bridge is Initializable, IBridge, IERC777Recipient, UpgradablePausable,
         return erc777ConverterAddr;
     }
 
+    function setBridgeReceiverStatus(
+        address receiverAddress,
+        bool status
+    ) external onlyOwner {
+        require(receiverAddress != address(0), "Cannot set zero address as bridge receiver");
+        isBridgeReceiver[receiverAddress] = status;
+        emit BridgeReceiverStatusChanged(receiverAddress, status);
+    }
```

Comment:
More often contracts don't limit possible interactions. Most projects assume that users know what they are doing and try to execute transactions unless it is insecure.

Reply:
Only verified by the owner SC can be part of the bridge-fastBTC bridges, otherwise the bridge is vulnerable to hacks

Reply:
i do not think that adding the user data for other contracts would make the bridge more vulnerable.
to my understanding, this check is rather fullfilling the purpose to find out if we should call a certain function on the receiver contract.